datafusion/core/tests/fuzz_cases/sort_fuzz.rs

```diff
@@ -54,7 +54,9 @@ async fn test_sort_10k_mem() {
 #[tokio::test]
 #[cfg_attr(tarpaulin, ignore)]
 async fn test_sort_100k_mem() {
```

Comment:
Can you please also add some fuzz tests for the cases you are coverting in this PR? Specifically:
1. `RecordBatch`es with String arrays
2. `RecordBatch`es that have multiple columns (and thus use the Row format)